src/main/kotlin/com/example/ipwhitelist/controller/AppController.kt

```diff
@@ -24,15 +23,15 @@ class AppController(
     @GetMapping
     fun findAppsByUserId(): ResponseEntity<List<EnhancedAppResponse>> {
         val authentication = SecurityContextHolder.getContext().authentication
-        val userPrincipal = userService.findByEmail(authentication.name)
-
-        val userApps = appService.getAppsByUserKey(userPrincipal!!.userId)
+        val userId = userService.findByEmail(authentication.name)!!.id
+        val userKey = "${UserTableKeyPrefix.USER}$userId"
```

Comment:
This should be `${UserTableKeyPrefix.USER.prefix}`, else it would be missing a hyphen

Reply:
good catch! fixing now